dart/lib/src/user_feedback.dart

```diff
@@ -0,0 +1,54 @@
+import 'protocol.dart';
+
+class UserFeedback {
```

Comment:
SentryUserFeedback so we avoid import coalision

Reply:
We also called it `UserFeedback` on [.NET](https://github.com/getsentry/sentry-dotnet/blob/main/src/Sentry/UserFeedback.cs) and [Java](https://github.com/getsentry/sentry-java/blob/main/sentry/src/main/java/io/sentry/UserFeedback.java). On Cocoa, it's `SentryUserFeedback` for Objective-C and `UserFeedback` for Swift. We did this\ because `UserFeedback` is a not so common class. So I would say `UserFeedback` is fine here.

Reply:
unless you use the library that @ueman developed himself, https://github.com/ueman/feedback/blob/master/lib/src/user_feedback.dart hehe
so `SentryUserFeedback` is the way to go, Dart is like objC for imports, so better safer than sorry